src/test/suite/oidc.test.ts

```diff
@@ -121,7 +121,7 @@ suite('OIDC Tests', function () {
 
     cluster = await MongoCluster.start({
       ...defaultClusterOptions,
-      version: '7.0.x',
+      version: '8.0.x',
```

Comment:
This was the actual fix - ubuntu-latest was bumped to 24.04, but 7.0.x only supports up to ubuntu 22, so we have to use a newer server.